docs/reference/settings/common-defs.asciidoc

```diff
@@ -111,6 +111,8 @@ Supported protocols with versions. Valid protocols: `SSLv2Hello`,
 the default is `TLSv1.3,TLSv1.2,TLSv1.1`. Otherwise, the default is
 `TLSv1.2,TLSv1.1`.
 +
+See <<jdk-tls-versions>>
```

Comment:
We could re-use the first sentence from the `jdk-tls-versions` page here to segue smoothly into the link, something like:
```
Elasticsearch relies on your JDK’s implementation of SSL and TLS, see also <<jdk-tls-versions>.
```
I feel it helps establishing the context better because we were talking about ES settings and why it now needs another docs page for JDK?